share/picongpu/pypicongpu/schema/species/operation/densityprofile/foil.Foil.json

```diff
@@ -0,0 +1,36 @@
+{
+    "$id": "https://registry.hzdr.de/crp/picongpu/schema/picongpu.pypicongpu.species.operation.densityprofile.foil.Foil",
```

Comment:
Well they are IDs allowing direct references for an individual schema. They happen to have the form of URLs since that is the expected format from the templating-engine, as usually they would be used to get a schema from the internal namespace of a website, or one from an secondary or external website.

They are built specifically to not resolve to a website and be part of the hzdr owned namespace to assure that no one is able to inject schemas into the code.

also see Issue #4088 for further information